### PR TITLE
Integrate CircleCI

### PR DESCRIPTION
**Pull request by liaujianjie**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:7.10
```
Comment (natealcedo) [inline .circleci/config.yml]: Let's do node 8 here

Comment (liaujianjie) [reply .circleci/config.yml]: So just `circleci/node:8`?

Comment (natealcedo) [reply .circleci/config.yml]: Yes, that should work. 

Jest uses `image: circleci/node:10` so just replacing it with 8 should work

**Comment by natealcedo** [comment]
Looks good!

**Pull request merged**